### DISCUSSION TITLE
is there any  SaveAll and LoadAll function

### DISCUSSION THREAD

**Discussion by AhmetNuri** [viniciussanchez/dataset-serialize#159] [Ideas]
Hello is there any method for save Save Structure and Data, Load Save Structure and data ?

**Comment by viniciussanchez** [comment]
![image](https://user-images.githubusercontent.com/16382981/177601284-d6068e6f-436e-4b6b-ab4a-8c1e93fe7b13.png)

**Comment by AhmetNuri** [comment]
Hello I test it before . this is only Structure post.
For remoute working 
 it should be in same array like that
[
{Structure : [bla bla ...]},
{datas : [  bla bla ... ]} 
]

]
Because querys are createing run time

**Reply by viniciussanchez** [reply]
I understood....
We don't have that yet.
